suite-native/graph/jest.config.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+    preset: '../../jest.config.base.js',
```

Comment:
Is this file necessary here? We are passing it with tests now.

Reply:
Guess not, but it's in nearly all packages in /suite-native so I am just following the convention we've estabilished. I think it was discussed before in your navigation PR? Not sure though. When tests get added (I guess quite soon since we're getting close to doing business logic) it will need to extend this config so I would leave it. If you think I should remove it, it should be removed from all other packages where there are not tests as well.

Reply:
I would prefer to remove tests stuff from packages where we don't have tests, but this package is definitely worth of having some tests because there is some logic (Time switching) that should be tested. So it would be nice to add them right now.

Reply:
You can use this lib https://testing-library.com it should pretty easy with that to add some basic tests for Graph component.

Reply:
Will be done in separate PR.